model/task_lifecycle.go

```diff
@@ -705,7 +705,14 @@ func RestartItemsAfterVersion(cq *commitqueue.CommitQueue, project, version, cal
 		}
 		if item.Version == version {
 			foundItem = true
-		} else if foundItem {
+		} else if foundItem && item.Version != "" {
```

Comment:
I don't think this makes the change you want it to. If item.Version is empty, then it returns at 704, so we should only get here if item.Version != ""

Reply:
good catch! I left the check here as a sanity check